src/mon/AuthMonitor.cc

```diff
@@ -1630,34 +1630,47 @@ bool AuthMonitor::prepare_command(MonOpRequestRef op)
 	}
       }
 
+      mon_cap_string += mon_cap_string.empty() ? "" : ", ";
```

Comment:
mon_cap_string should just be 'allow r fsname= \<fsname\>' irrespective of the cap value, which at least 'r'.

Reply:
"which at least 'r'"?

Reply:
the cap value is at least 'r'

Reply:
Changed. Thanks!